libraries/plugins/custom_operations/custom_operations_plugin.cpp

```diff
@@ -126,14 +128,25 @@ void custom_operations_plugin::plugin_set_program_options(
    boost::program_options::options_description& cfg
    )
 {
+   cli.add_options()
+         ("custom-operations-start-after-block", boost::program_options::value<uint32_t>()->default_value(45000000),
```

Comment:
I think "custom-operations-start-block" is better. The use of "after" is a bit misleading.